### PR TITLE
Added braille graph theme 

### PR DESCRIPTION
**Pull request by garmoned**
With braille you can draw vertical lines to watch your stonks go down fast 
difference

### CODE REVIEW

File: graph/graph.go [hunk 1]
```diff
@@ -186,27 +171,137 @@ check:
...
 			}
+			out += "\033[0m┃"
+			out += "\n"
 		}
-		out += "\033[0m┃"
-		out += "\n"
+
 	}
+
+	//
```
Comment (ericm) [inline graph/graph.go:303]: Remove empty comment

File: graph/graph.go [hunk 2]
```diff
@@ -33,71 +36,53 @@ func borderHorizontal(out *string, width int) {
...
-		chart.Length = width / 3
-	}
-	if chart.Length > width {
-		mod := 2 * (chart.Length / width)
-		chartTemp := make([]*api.Bar, 0)
-		for i, bar := range chart.Bars {
-			if (i+1)%mod == 0 {
-				chartTemp = append(chartTemp, bar)
-			}
+func GenerateBraille(chart *api.Chart, width int, height int) string {
```
Comment (ericm) [inline graph/graph.go:39]: I'm unsure why there's a need to delegate generation of the braile theme to a new function. The rest of the themes just replace the relevant characters in a condtional as seen [here](https://github.com/ericm/stonks/blob/1e9523c54994076b7711e0325329fd02b7594290/graph/graph.go#L107). Could it be refactored to do the same?

File: graph/graph.go [hunk 3]
```diff
@@ -254,4 +349,5 @@ checkFooter:
 	borderHorizontal(&out, width+maxSize+3)
 	out += "┛\n"
 	return out, nil
+
```
Comment (ericm) [inline graph/graph.go:352]: Unecessary breakline

**Review by ericm — CHANGES_REQUESTED** [review]

File: graph/graph.go (hunk 2 above)
Comment (garmoned) [reply graph/graph.go:39]: The braille graph is drawn in an entirely different way from the other graph themes so I don't think the same result can be achieved with just a character swap.

Comment (ericm) [reply graph/graph.go:39]: Could you add it to the flag `-t`. I'm also not sure about delegating to an unmaintaned fork to draw braille considering the `graph` package is supposed to contain the generation code.

File: graph/graph.go [hunk 4]
```diff
@@ -186,27 +171,137 @@ check:
...
-	for i, slc := range matrix {
-		out += "┃"
-		price := chart.High.Sub(increment.Mul(decimal.NewFromInt(int64(i)))).StringFixed(2)
-	checkLen:
-		if len(price) < maxSize {
-			price = " " + price
-			goto checkLen
+	out += "\n"
+
+	if chartTheme == Braille {
```
Comment (ericm) [inline graph/graph.go:249]: Using the package also introduces extra logic here some of which isn't 100% DRY. Could you implement the packages functionality in a more cohesive way inside the package? Cheers :)

File: graph/graph.go (hunk 2 above)
Comment (garmoned) [reply graph/graph.go:39]: Is the use of drawille-go a complete deal breaker? If so I can attempt to re implement the drawing without a library.

**Pull request closed without merging**